index.js

```diff
@@ -31,9 +31,10 @@ function sseMiddleware(__, res, next) {
     if (id) {
       res.write(`id: ${id}\n`);
     }
-
+    if (evt) {
```

Comment:
Or probably it would be better to support both construction:

`res.sse(data, id)`
and
`res.sse(eventName, data, id)`